src/taskgraph/run-task/run-task

```diff
@@ -1078,6 +1084,51 @@ def _display_python_version():
         b"Python version: "+ bytes(platform.python_version(), 'utf-8')
     )
 
+def _display_python_version_in_sub_process(subprocess: subprocess):
```

Comment:
Refactor: this function can be made much simpler with a for-loop that loops over `("python", "python2", "python3")`.

Reply:
I understand and I tought about doing it. but the "business" objectives are different. for example if we want to treat differently the case when we have a python 2.x involved. this is why i kept them separated. I can of course refactor.

Reply:
Got it! Could you expand on the py2.x case you foresee? Maybe there's something I missed here 🙂